extensions/amp-social-share/1.0/social-share.js

```diff
@@ -28,6 +28,17 @@ import {useResourcesNotify} from '../../../src/preact/utils';
 const DEFAULT_WIDTH = 60;
 const DEFAULT_HEIGHT = 44;
 const NAME = 'SocialShare';
+const DEFINED_PROPERTIES = new Set([
```

Comment:
Nit: Just using an array here will save us the polyfill bytes. This is small enough that an array's `includes` check will always be faster than the set's `has`.

Reply:
Thanks! Updated